composer.json

```diff
@@ -33,9 +33,9 @@
         "psr/log": "^1.0",
         "swiftmailer/swiftmailer": "^6.1",
         "symfony/http-foundation": "^3.3",
-        "symfony/psr-http-message-bridge": "^1.0",
+        "symfony/psr-http-message-bridge": "~v1.1.0",
```

Comment:
This pinning is not needed to fix the error message?

Reply:
It is. One deprecation error comes with version `v1.2.0`…

Reply:
Ah, ok, now i see ^^
Thats the code that i fixed at the weekend and didn't create a PR because it was not tested ;)
-> Will create one soon

Reply:
Fyi, on my Gitlab fork all CI jobs pass for this PR.

Reply:
But its a preventable workaround :P